dockerfiles/prod/Dockerfile-elm

```diff
@@ -0,0 +1,14 @@
+FROM node:6.9.1
+RUN npm install -g yarn
+WORKDIR /code
+COPY elm-package.json elm-package.json
+COPY package.json package.json
+COPY gulpfile.js gulpfile.js
+COPY jarbas jarbas
+COPY contrib/copyassets.sh contrib/copyassets.sh
+RUN chmod +x contrib/copyassets.sh
+RUN yarn install
+VOLUME /assets
+RUN yarn assets
+CMD ["./contrib/copyassets.sh"]
+
```

Comment:
I don't like the copies made in dev environment, is a process unnecessary because we are mounting the shared volumes in containers.
When in prod environment, the copies are good because of the performance improvement, I guess.